backend/internal/server/apis/stats/penalty.go

```diff
@@ -0,0 +1,64 @@
+package stats
+
+import (
+	"encoding/json"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/db"
+	"github.com/jak103/powerplay/internal/models"
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/jak103/powerplay/internal/server/services/auth"
+	"github.com/jak103/powerplay/internal/utils/locals"
+	"github.com/jak103/powerplay/internal/utils/responder"
+)
+
+func init() {
+	//apis.RegisterHandler(fiber.MethodGet, "/penaltyTypes", auth.Public, getPenaltyTypes)
+	apis.RegisterHandler(fiber.MethodGet, "/penalties", auth.Public, getPenaltiesHandler)
+	apis.RegisterHandler(fiber.MethodPost, "/penalties", auth.Public, postPenaltyHandler)
+}
+
+func getPenaltiesHandler(c *fiber.Ctx) error {
+	log := locals.Logger(c)
+	log.Info("Handling getting all penalties")
+	db := db.GetSession(c)
+	penalties, err := db.GetPenalties()
+	if err != nil {
+		log.WithErr(err).Alert("Failed to get all penalties from the database")
```

Comment:
⚠️ Does this error go to the FE? If so we probably don't want to be this specific and have more generic error messages -- but this information would be good to log to the backend.

Reply:
The error should be logged to the backend, never to the frontend. However I just tested this and it's not working as it should. I'll address that.